docsrc/create_docs.sh

```diff
@@ -1,7 +1,11 @@
 #!/bin/bash
 rm -rf source/apidoc
-sphinx-apidoc -o source/apidoc ../producer
-python3 edit_apidoc_modules.py
+sphinx-apidoc -o source/apidoc ../python/love ../python/love/producer/test_utils/*
+python edit_apidoc_modules.py
 rm -rf ../docs/doctrees
 rm -rf ../docs/html
 make html
+touch ../docs/.nojekyll
+echo "<meta http-equiv='refresh' content='0; url=./html/index.html' />" > ../docs/index.html
+chown -R 1000:1000 ../docs
```

Comment:
This was because we had a problem with our local development containers, we needed to make that change. Although now this was moved to the Jenkinsfile instructions 👍 .